docs/guides/main.rst

```diff
@@ -290,9 +290,19 @@ You can always get help using the ``beet help`` command. The plain ``beet help``
 command lists all the available commands; then, for example, ``beet help
 import`` gives more specific help about the ``import`` command.
 
+Core Concepts
+-------------
+
+* *Library*: Is the beets database model only meant to catalog users music , which means that it only represents links to music files. The library is commposed by @items(songs) and @albums. The metadata album is shared by the track groups. Each item belongs to an album, otherwise is a @singleton. Users can import songs with the ``beet import`` command. 
+* *Queries*: Represent the data subsets that the user wants to retrieve from the library. 
+* *Paths and Destinations*: Each item has a @destination, which is the path it would reside at. In its default setup, the beet import command copies items to their destination as they are imported. 
+* *Auto-Tagging*: Beets used two auto-tagging workflows, one for Albums and one for Items (i.e., singletons). This ones retrieve metadata provider by MusicBrainz, Beatport etc.
```

Comment:
used -> uses
In both cases, metadata is retrieved from providers such as MusicBrainz.